authorization-server/package.json

```diff
@@ -30,6 +30,11 @@
     "test:e2e": "cd test && jest --config ./jest-e2e.json"
   },
   "dependencies": {
+    "@energyweb/credential-governance": "^1.0.1-alpha.242.0",
```

Comment:
```suggestion
    "@energyweb/credential-governance": "^2.1.0",
```

Reply:
I needed to downgrade to this version otherwise, the TS compiler throws errors.

Reply:
Aah, I see. Is it possible to upgrade TS? I guess it would be good to do so in order to utilise the functionalities at its best and be consistent with all other repos?

Reply:
If you revert 51f234f3a213f929da08d30e6e45674a076aeb84 commit, code does not compile.

Reply:
Once PDA gets the latest update for `ew-credentials`, this would resolve I think.

Reply:
Maybe PDA just should export dependencies that are required to initialize it?  There would be no need to struggle with versions anymore. What do you think?

Reply:
Yeah, I think we can do this.

Reply:
It would be a very clean solution for anyone who integrates PDA into their code. We can also think about having all these dependencies added to the constructor in the last version, to be instantiated by the PDA if not provided explicitly.